### PR TITLE
Updates for BGZIPTABIX

### PR DESCRIPTION
**Pull request by DLBPointon**
## PR checklist

Closes ##3423

I've added an option for the module to output a CSI file. 

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [-] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [-] If necessary, include test data in your PR.
- [-] Remove all TODO statements.
- [-] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [-] Add a resource `label`
- [-] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]
Loving this

**Comment by maxulysse** [comment]
Can you check the md5sum from GHA, they don't seem to match

File: modules/nf-core/tabix/bgziptabix/main.nf [hunk 1]
```diff
@@ -5,13 +5,14 @@ process TABIX_BGZIPTABIX {
     conda "bioconda::tabix=1.11"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/tabix:1.11--hdfd78af_0' :
-        'biocontainers/tabix:1.11--hdfd78af_0' }"
+        'quay.io/biocontainers/tabix:1.11--hdfd78af_0' }"
```
Comment (nvnieuwk) [inline modules/nf-core/tabix/bgziptabix/main.nf:8]: You don't have to put the `quay.io` here anymore (this is now the default repo in the nf-core pipelines)

**Comment by maxulysse** [comment]
@DLBPointon and what happend to the md5sums?
Did you check them?

**Comment by DLBPointon** [comment]
Hi, 
Sorry, i should have checked in with you. I rush merged this morning.
On my end they are matching, docker and singularity match the expected. Conda on the other hand does not seem to want to match.

**Pull request merged**